src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -160,20 +142,24 @@ public double getAverageDistance() {
     return (getLeftDistance() + getRightDistance()) / 2;
   }
 
-  /**
-   * resets the encoder
-   */
-  public void resetEncoders() {
-    m_leftEncoder.reset();
-    m_rightEncoder.reset();
+  public void resetDriveTrain() {
+    resetDriveTrain(0.0, 0.0, 0.0);
   }
 
-  /**
-   * resets the gyro and the odometry
+   /**
+   * Resets the odometry and sets the robot to the inputted position.
+   * Also resets encoders and gyro
+   * 
+   * @param xPos  the x position of the robot in inches
+   * @param yPos  the y position of the robot in inches
+   * @param angle the angle of the robot in degrees (WPILIB Format, degrees
+   *              counterclockwise with 0 being straight ahead)
```

Comment:
If you wish to shorten this, you can say "the angle of the robot in degrees CCW from forward"

Reply:
resolved in 0bf8ea1